newIDE/app/src/Profile/Subscription/PlanCard.js

```diff
@@ -67,62 +62,117 @@ const getPlanPrice = ({
   return prices;
 };
 
+const getPlanIcon = (plan: PlanDetails): React.Node => {
+  switch (plan.planId) {
+    case 'gdevelop_silver':
+    case 'gdevelop_indie': // legacy
+      return <Silver fontSize="large" style={{ width: 65, height: 65 }} />;
+    case 'gdevelop_gold':
+    case 'gdevelop_pro': // legacy
+      return <Gold style={{ width: 65, height: 65 }} />;
+    // TODO: Add icons for other plans.
+    case 'gdevelop_education':
+    case 'gdevelop_startup':
+    case 'gdevelop_enterprise':
+    default:
+      return <GDevelopGLogo style={{ width: 25, height: 25, padding: 20 }} />;
```

Comment:
Should those values be stored in constants with an operation like `WIDTH - 2*PADDING`?

Reply:
yep, makes sense, will do